Modules/xSharePoint/DSCResources/MSFT_xSPFarmSolution/MSFT_xSPFarmSolution.schema.mof

```diff
@@ -0,0 +1,26 @@
+/*
+**Description**
+
+This resource will deploy a farm solution to a farm. A version will be stored in the property bag to determin later if the correct version is installed.
+
+**Example**
+
+    xSPFarmSolution SampleWsp
+    {
+        Name                 = "MySolution.wsp"
+		LiteralPath          = "C:\src\MySolution.wsp"
+		Version              = "1.0.0"
+        PsDscRunAsCredential = $InstallAccount
+    }
+*/
+[ClassVersion("1.0.0.0"), FriendlyName("xSPFarmSolution")]
+class MSFT_xSPFarmSolution : OMI_BaseResource
+{
+    [Key] string Name;
+    [Required] string LiteralPath;
+    [Write] string WebApplications[];
+    [Write] string Ensure;
+	[Write] string Version;
+	[Write] Boolean Deployed;
+    [Write, EmbeddedInstance("MSFT_Credential")] String InstallAccount;
```

Comment:
@TravisEz13 Yes, this is by design [as per our guidance](https://github.com/powershell/xSharePoint/wiki/Remote%20sessions%20and%20the%20InstallAccount%20variable) so we can support PowerShell 4 while offering a better approach for PowerShell 5. We have a unit test that makes sure it is optional where it is used.